WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -181,36 +181,11 @@ protected override async Task ActionAsync(CancellationToken cancel)
 				AllFeeEstimateArrived?.Invoke(this, allFeeEstimate);
 			}
 		}
-		catch (OperationCanceledException)
+		catch (HttpRequestException)
```

Comment:
`TorConnectionException` was removed as a condition because it the user turns off Tor, there won't be `TorConnectionException` as an `InnerException`.

Reply:
That is related to the status display at the right bottom corner. I don't know any issues with that.